azurerm/resource_arm_operational_insight_workspace_test.go

```diff
@@ -0,0 +1,208 @@
+package azurerm
```

Comment:
(as above) could we rename this to `resource_arm_log_analysis_workspace_test.go`?

Reply:
Done